spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -46,83 +46,79 @@ MicroProfile applications that make use of APIs or versions not mentioned below
 
-JSON-B is leveraged as part of MicroProfile 2.0 to provide standard APIs for binding JSON documents to Java code.
-Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-B is leveraged as part of MicroProfile 4.0 to provide standard APIs for binding JSON documents to Java code.
+Use of implementations beyond JSON-B 1.0 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonb-spec
- - https://jcp.org/en/jsr/detail?id=367
+ - https://jakarta.ee/specifications/jsonb/1.0/jsonb-spec-1.0.html
+ - https://github.com/eclipse-ee4j/jsonb-api
 
-[[javaee-jsonp]]
-=== JSON-P 1.1
+[[jakarta-jsonp]]
+=== Jakarta JSON-P 1.1
 
-JSON-P is leveraged as part of MicroProfile 2.0 to provide standard APIs for processing JSON documents.
-Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+JSON-P is leveraged as part of MicroProfile 4.0 to provide standard APIs for processing JSON documents.
+Use of implementations beyond JSON-P 1.1 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/jsonp/
- - https://jcp.org/en/jsr/detail?id=374
+ - https://jakarta.ee/specifications/jsonp/1.1/jsonp-spec-1.1.html
+ - https://github.com/eclipse-ee4j/jsonp
 
-[[javaee-common-annotations]]
-=== Common Annotations 1.3
+[[jakarta-annotations]]
+=== Jakarta Annotations 1.3
 
-Common Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Java(TM) EE platforms.
-Use of implementations beyond Common Annotations 1.3 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+Jakarta Annotations provides annotations for common semantic concepts across a variety of individual technologies in the Java(TM) SE and Jakarta EE platforms.
+Use of implementations beyond Jakarta Annotations 1.3 in MicroProfile is allowed, however, not required.
 
- - https://javaee.github.io/glassfish/
- - https://jcp.org/en/jsr/detail?id=250
+ - https://jakarta.ee/specifications/annotations/1.3/annotations-spec-1.3.html
+ - https://github.com/eclipse-ee4j/common-annotations-api
 
 [[mp-config]]
-=== Eclipse MicroProfile Config 1.4
+=== Eclipse MicroProfile Config 2.0
 
 Eclipse MicroProfile Config provides applications and microservices means to obtain configuration properties through several environment-aware sources both internal and external to the application and made available through dependency injection or lookup.
 
  - http://microprofile.io/project/eclipse/microprofile-config
- - https://github.com/eclipse/microprofile-config/releases/tag/1.4
-
-The Eclipse MicroProfile Config API is currently in the JSR process and is actively seeking adoption stories, feedback, and Expert
-Group members.
-
- - https://jcp.org/en/jsr/detail?id=382
+ - https://github.com/eclipse/microprofile-config/releases/tag/2.0
```

Comment:
The final 2.0 tag is not available yet.  We're still at a Milestone release...  This comment relates to my previous comment.

Both of these comments have a ripple effect to all of the following MP API descriptions and associated versions.

Reply:
Ok, this is a bit of admitted double-speak for me. I provide the milestone release links in the architecture doc but intentionally point to "final" tags in the required-apis doc.  I've done this in the past as well. I can fix this but hoping to avoid it for less work later. Fix anyway?

Reply:
I think we need to...

Reply:
Will fix.